exchangelib/ewsdatetime.py

```diff
@@ -97,7 +97,10 @@ def ewsformat(self):
         if not self.tzinfo:
             raise ValueError('EWSDateTime must be timezone-aware')
         if self.tzinfo.zone == 'UTC':
-            return self.strftime('%Y-%m-%dT%H:%M:%SZ')
+            try:
+                return self.strftime('%Y-%m-%dT%H:%M:%SZ')
+            except ValueError:
+                return '{}-{}-{}T{}:{}:{}'.format(self.year, self.month, self.day, self.hour, self.minute, self.second)
```

Comment:
I would add a comment saying that `.strftime()` throws a ValueError when `year < 1900`, so someone doesn't wonder why we could still print this if there was a ValueError.

Reply:
Also do we need/want the trailing `Z`?

Reply:
Hmm, another thought, should we just always do the manual string format? It should work in all cases right? And then we could add a comment saying why we have to manually format it instead of using `.strftime()`

Reply:
@Gerdie the trailing Z indicates that the time is in UTC. When a time is not in UTC then it would have the offset so `2018-04-05T12:30-02:00` -- more info here [https://en.wikipedia.org/wiki/ISO_8601#Times] under Coordinated Universal Time

@hallamoore `.strftime()` has nicer handling of the printed datetimes such that they look more like datetimes. Each of `%Y`, `%m`, `%d`, etc are turned into standardized, zero padded versions of that number. I can switch to just use barebones printing like line 103 but I'm not sure this is better.

Reply:
Ah, nah this is fine then. To @Gerdie's point, I think we do want the `Z`, but it's missing in L103